<ui version="4.0" >
 <class>AutoxWindow</class>
 <widget class="QWidget" name="AutoxWindow" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>182</width>
    <height>255</height>
   </rect>
  </property>
  <property name="focusPolicy" >
   <enum>Qt::StrongFocus</enum>
  </property>
  <property name="acceptDrops" >
   <bool>false</bool>
  </property>
  <property name="windowTitle" >
   <string>3dmod AutoContour</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" >
   <item>
    <widget class="QGroupBox" name="groupBox" >
     <property name="title" >
      <string>Contrast</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout" >
      <property name="leftMargin" >
       <number>5</number>
      </property>
      <property name="topMargin" >
       <number>2</number>
      </property>
      <property name="rightMargin" >
       <number>5</number>
      </property>
      <property name="bottomMargin" >
       <number>5</number>
      </property>
      <item>
       <widget class="QRadioButton" name="regularRadioButton" >
        <property name="focusPolicy" >
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="toolTip" >
         <string>Display with normal black/white contrast settings</string>
        </property>
        <property name="text" >
         <string>Regular</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="highRadioButton" >
        <property name="focusPolicy" >
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="toolTip" >
         <string>Display image as black/white using threshold</string>
        </property>
        <property name="text" >
         <string>High</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="sliderFrame" >
     <property name="frameShape" >
      <enum>QFrame::Panel</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Plain</enum>
     </property>
     <property name="lineWidth" >
      <number>0</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="altMouseBox" >
     <property name="focusPolicy" >
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="toolTip" >
      <string>Set mouse buttons to fill, insert, and delete pixels</string>
     </property>
     <property name="text" >
      <string>Alternate mouse</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="diagonalsBox" >
     <property name="focusPolicy" >
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="toolTip" >
      <string>Connect pixels touching only on corners when filling</string>
     </property>
     <property name="text" >
      <string>Follow diagonals</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="spacing" >
      <number>6</number>
     </property>
     <property name="margin" >
      <number>0</number>
     </property>
     <item>
      <widget class="QPushButton" name="fillButton" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize" >
        <size>
         <width>35</width>
         <height>32767</height>
        </size>
       </property>
       <property name="focusPolicy" >
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="toolTip" >
        <string>Select area based on threshold</string>
       </property>
       <property name="text" >
        <string>Fill</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="clearButton" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize" >
        <size>
         <width>35</width>
         <height>32767</height>
        </size>
       </property>
       <property name="focusPolicy" >
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="toolTip" >
        <string>Clear filled area</string>
       </property>
       <property name="text" >
        <string>Clear</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buildButton" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize" >
        <size>
         <width>35</width>
         <height>32767</height>
        </size>
       </property>
       <property name="focusPolicy" >
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="toolTip" >
        <string>Build contour around filled area (hot key b)</string>
       </property>
       <property name="text" >
        <string>Build</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="nextButton" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize" >
        <size>
         <width>35</width>
         <height>32767</height>
        </size>
       </property>
       <property name="focusPolicy" >
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="toolTip" >
        <string>Fill area on next section (hot key a)</string>
       </property>
       <property name="text" >
        <string>Next</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="spacing" >
      <number>6</number>
     </property>
     <property name="margin" >
      <number>0</number>
     </property>
     <item>
      <widget class="QPushButton" name="smoothButton" >
       <property name="maximumSize" >
        <size>
         <width>50</width>
         <height>32767</height>
        </size>
       </property>
       <property name="focusPolicy" >
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="toolTip" >
        <string>Smooth by expanding then shrinking (hot key u)</string>
       </property>
       <property name="text" >
        <string>Smooth</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="shrinkButton" >
       <property name="maximumSize" >
        <size>
         <width>50</width>
         <height>32767</height>
        </size>
       </property>
       <property name="focusPolicy" >
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="toolTip" >
        <string>Shrink filled area</string>
       </property>
       <property name="text" >
        <string>Shrink</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="expandButton" >
       <property name="maximumSize" >
        <size>
         <width>50</width>
         <height>32767</height>
        </size>
       </property>
       <property name="focusPolicy" >
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="toolTip" >
        <string>Expand filled area</string>
       </property>
       <property name="text" >
        <string>Expand</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="Line1" >
     <property name="frameShape" >
      <enum>QFrame::HLine</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Sunken</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="spacing" >
      <number>6</number>
     </property>
     <property name="margin" >
      <number>0</number>
     </property>
     <item>
      <widget class="QPushButton" name="doneButton" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize" >
        <size>
         <width>60</width>
         <height>32767</height>
        </size>
       </property>
       <property name="focusPolicy" >
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="toolTip" >
        <string>Close dialog box</string>
       </property>
       <property name="text" >
        <string>Done</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="helpButton" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize" >
        <size>
         <width>60</width>
         <height>32767</height>
        </size>
       </property>
       <property name="focusPolicy" >
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text" >
        <string>Help</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <resources/>
 <connections>
  <connection>
   <sender>altMouseBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>AutoxWindow</receiver>
   <slot>altMouse(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>diagonalsBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>AutoxWindow</receiver>
   <slot>followDiagonals(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>fillButton</sender>
   <signal>clicked()</signal>
   <receiver>AutoxWindow</receiver>
   <slot>fillPressed()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>clearButton</sender>
   <signal>clicked()</signal>
   <receiver>AutoxWindow</receiver>
   <slot>clearPressed()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buildButton</sender>
   <signal>clicked()</signal>
   <receiver>AutoxWindow</receiver>
   <slot>buildPressed()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>nextButton</sender>
   <signal>clicked()</signal>
   <receiver>AutoxWindow</receiver>
   <slot>nextPressed()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>smoothButton</sender>
   <signal>clicked()</signal>
   <receiver>AutoxWindow</receiver>
   <slot>smoothPressed()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>shrinkButton</sender>
   <signal>clicked()</signal>
   <receiver>AutoxWindow</receiver>
   <slot>shrinkPressed()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>expandButton</sender>
   <signal>clicked()</signal>
   <receiver>AutoxWindow</receiver>
   <slot>expandPressed()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>doneButton</sender>
   <signal>clicked()</signal>
   <receiver>AutoxWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>helpButton</sender>
   <signal>clicked()</signal>
   <receiver>AutoxWindow</receiver>
   <slot>helpPressed()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
